tests/testsuite.src/run_misc.at

```diff
@@ -10234,6 +10243,8 @@ CAPI (void *p1, ...)
       printf("Line%3d: ",k);
    }
    printf ("CALL with %d parameters\n",nargs);
+   fflush (stdout);
+   fflush(stdout);
```

Comment:
the one with the space before the `(` is fine (it isn't that way in line 10602 btw); but we shouldn't need two calls...
..

Reply:
The duplicate `fflush` seems to be a merge accident (had made the change both in 3.x and in this PR, then merged the commit from 3.x, which did not generate a conflict...). Fixing this and the spaces.